README.md

```diff
@@ -23,6 +23,7 @@ scrubbing](http://sjp.pwn.pl/sjp/;2527372). It is pronounced as *shoorubooru*.
 - Pools and pool categories
 - Duplicate detection
 - Post rating and favoriting; comment rating
+- Sorting by true chronological order (through source file's last modified time)
```

Comment:
I feel like this would be covered under "Rich search system". Don't really see why we need to highlight this search token over all the other ones.

Reply:
Okay. It's just that it is not really found in many other boorus.